lib/daru/core/group_by.rb

```diff
@@ -267,11 +270,11 @@ def apply_method method_type, method
         multi_index = multi_indexed_grouping?
         rows, order = [], []
 
-        @groups.each do |group, indexes|
+        @groups.each do |_group, indexes|
```

Comment:
It is Rubocop-foced agreement for "not used argument". You can use `_` prefix or  name the whole argument `_` to mark it as skipped.